docs/ocean/leap_authorization.rst

```diff
@@ -10,40 +10,92 @@ and higher enable Leap\ |tm| authentication using the secure
 `OAuth 2.0 <https://oauth.net/2/>`_ industry standard. As described in the
 :ref:`ocean_sapi_access_basic` section, you require a Solver API (:term:`SAPI`)
 token from your Leap account to submit problems to Leap
-:term:`solvers <solver>`. The following procedure authorizes the Ocean software
-access to your account in the Leap service and enables storing your SAPI token
-in your development environment. Ocean software authorization to access the Leap
-service persists across subsequent development sessions for this development
-environment.
+:term:`solvers <solver>`.
 
-.. note:: The following `OAuth-based <https://oauth.net/2/>`_ procedure is
-    provided for convenience but you also have the option of manually copying
-    your SAPI token from the dashboard in the Leap service. The
-    :ref:`ocean_sapi_access_basic` section provides more information.
+Authorization Procedure
+=======================
 
-This procedure uses the :ref:`Ocean software CLI <ocean_dwave_cli>` commands
-that you enter into your terminal. You can see help documentation for these
-commands and all their options with the CLI's ``--help`` option.
+The following procedure authorizes the Ocean software access to your account in
+the Leap service and enables storing your SAPI token in your development
+environment. Ocean software authorization to access the Leap service persists
+across subsequent development sessions for this development environment.
+
+This procedure uses the :ref:`Ocean software CLI <ocean_dwave_cli>`
+commands\ [#]_ that you enter into your terminal.
```

Comment:
Broken ref. It shows `[1]` in the [built docs](https://output.circle-artifacts.com/output/job/6efee9a3-37d6-41d1-a4f8-c93a5630cf49/artifacts/0/docs/_build/html/ocean/leap_authorization.html), but it's linked with a hidden footnote on the second tab below. Which is also irrelevant for this "CLI commands" ref.

Reply:
Fixed, it was to the footnote at the bottom but was hijacked by the new footnote ref. Removed the footnote.